ui/app/controllers/vault/cluster/auth.js

```diff
@@ -67,8 +67,10 @@ export default Controller.extend({
     }
     transition.followRedirects().then(() => {
       if (isRoot) {
+        this.auth.set('isRootToken', true);
         this.flashMessages.warning(
-          'You have logged in with a root token. As a security precaution, this root token will not be stored by your browser and you will need to re-authenticate after the window is closed or refreshed.'
+          'You have logged in with a root token. As a security precaution, this root token will not be stored by your browser and you will need to re-authenticate after the window is closed or refreshed.',
+          ENV.environment !== 'development' ? { sticky: true } : {}
```

Comment:
Is making the flash message sticky necessary?

Reply:
This is the alternative we decided in UI sync last week instead of implementing a modal (since that seemed the _most_ disruptive)

But - I'd agree that the overall issue is **I think** when users are logged out unexpectedly which happens 1. changing namespaces and 2. when we refresh internally 

We could probably revert the stick flash message, or make it appear a little bit longer, in favor of tracking down and adding a warning to the "sneaky" times (problem 2 above) we log people out. What do you think?

Reply:
I thought the initial intent of the modal was to warn people after they have clicked a namespace link, before we transition them, to warn that they were about to be logged out due to using a root token. That seems to be where the most complaints have been, which corresponds to your `1` above. 

For refreshing internally, it probably would be good to have at least an internal map of when that happens. I haven't heard as many complaints about that but if the longer-lived flash message was discussed to handle that case, that makes sense to me

Reply:
We discussed alternatives to the modal during ui sync last week because there was confusion on the original ask. 

That's helpful context that most of the complaints were related to changing namespaces. I agree - the namespace picker feels sufficient. 

We didn't discuss a longer-lived flash message, but perhaps that can be revisited (along with internal mapping of refreshing) if more user complaints come in after this change

Reply:
Amazing thanks Claire + Chelsea for the feedback! I just updated the changelog + removed the sticky from the flash message. If either one of you has time, could you take a look? :)